.github/workflows/git_fixup.yml

```diff
@@ -0,0 +1,12 @@
+name: Git Checks
+
+on: [pull_request]
+
+jobs:
+  block-fixup:
```

Comment:
what is this fixup for ?

Reply:
This is to avoid accidentally merging commits that were created with `git commit --fixup` or `git commit --squash`, which happened to me a few times in the past.  See [here](https://fle.github.io/git-tip-keep-your-branch-clean-with-fixup-and-autosquash.html), in case you're not familiar with `--fixup`.